libs/ast_generic/AST_generic.ml

```diff
@@ -2427,3 +2427,15 @@ class virtual ['self] map_legacy =
       let skind = self#visit_stmt_kind env x.s in
       { x with s = skind }
   end
+
+type dependency = {
```

Comment:
probably better out of libs/, in src/core/Supply_chain.ml or src/core/ProjectDependency.ml

Reply:
or PackageDependency.ml maybe given the field names.